server.js

```diff
@@ -93,6 +93,10 @@ app.prepare().then(() => {
   server.use("/api/person", personRouter);
   server.use("/api/contribution", contributionRouter);
   server.use("/api/live", liveRouter);
+  server.post("/api/scan", (req, res) => {
```

Comment:
Temporary route, will clean this up with a future PR

Reply:
sounds good